client/components/sites-dropdown/style.scss

```diff
@@ -100,24 +101,6 @@
 			}
 		}
 	}
-
-	// Highlight selected site
-	&.is-selected,
```

Comment:
The behaviour of the sites dropdown is a bit awkward now, it should highlight the currently selected site when you open the dropdown, not the first in the list.